vcluster/_partials/quick-start-guide/view-host-resources.mdx

```diff
@@ -57,10 +57,23 @@ To verify this, perform these steps:
 
       You can see from the output that the the two NGINX pods exist in the host cluster. To prevent collisions, the pod names and their namespaces are rewritten by vCluster during the sync process from the virtual cluster to the host cluster.
 
-      
+
 :::info vCluster Control Plane
 The vCluster `my-cluster-0` pod is also known as the vCluster control plane.
+
+A virtual cluster is deployed as a
+[StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/),
```

Comment:
I forgot the use case of when this happens, but there are a handful of options that change the control plane pods from a statefulset to a deployment. 

@FabianKramm Can you clarify those use cases?

Reply:
Also related 

When deploying [vCluster in HA mode](https://www.vcluster.com/docs/vcluster/deploy/topologies/high-availability), I'm getting deployment and statefulset rather than 2 statefulsets.  Is this a bug or we should adjust the docs (and vcluster.yaml)  have stateful set for `etcd` and deployment for `vCluster`?

Reply:
The vcluster.yaml won't change (as I've already discussed that), but more that we should adjust the docs. 

In default use case it deploys a statefulSet, but there are use cases when it becomes a deployment. @Piotr1215 Can you follow up with Fabian to understand when it's statefulset vs. deployment?

Reply:
Basically whenever no local persistent storage is needed the statefulset becomes a regular deployment. This is the case for external backing stores such as external etcd or external database or if persistence is disabled.

Reply:
@Piotr1215 This still isn't correct from what I understand what Fabian is saying vs. what is written. Let's talk together on this one.

Reply:
@deniseschannon there are actually two versions of the same file, one for the `main` branch and one for the latest stable release, and I updated only one. Fixed it now, please re-review.